ast/parser.go

```diff
@@ -102,40 +103,11 @@ type ParserOptions struct {
 	AllFutureKeywords      bool
 	FutureKeywords         []string
 	SkipRules              bool
-	JSONOptions            *JSONOptions
+	JSONOptions            *marshal.JSONOptions
 	unreleasedKeywords     bool // TODO(sr): cleanup
 	generalRuleRefsEnabled bool
 }
 
-// JSONOptions defines the options for JSON operations,
```

Comment:
```go
type JSONOptions = marshal.JSONOptions
```
? 😉

Reply:
Ahh, hmm. I guess we don't want to remove the ast.JSONOptions type...?

Reply:
It's fine. I doubt it has seen many users, it's been introduced by you very recently, hasn't it?

Reply:
Yes, it's a relatively new addition.